### DISCUSSION TITLE
没有办法打开ioc文件

### DISCUSSION THREAD

**Discussion by F1u0rite** [peng-zhihui/HelloWord-Keyboard#75] [Q&A]
我6.61的CubeMX打开ioc会报错，不管是点continue还是migrate都是会报错的，没有办法回退到6.2.1该怎么办呀

**Comment by cyijun** [comment]
官网有历史版本下载的

**Comment by Wangzm007** [comment]
我是卸载的新版本重新在下载安装的6.2.1版本,好像这个软件就是新版本不兼容旧版本.
